crates/holochain/tests/tests/countersigning.rs

```diff
@@ -127,7 +127,11 @@ async fn retry_countersigning_commit_on_missing_deps() {
     // but before creating any data, disable publish and recent gossip.
     // Now the only way peers can get data is through get requests!
     for conductor in conductors.iter_mut() {
-        conductor.update_config(|c| SweetConductorConfig::from(c).historical_only().into());
+        conductor.update_config(|c| {
```

Comment:
This isn't actually right. We need to be disabling countersigning publishing, not general publishing, with those being split up. Bit of a funky test as it stands.